cypress/pom/loginPageObject.js

```diff
@@ -0,0 +1,57 @@
+class LoginPage {
+
+    visit(){
+        cy.visit('/')
+    }
+
+    getLoginButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getUserName(userName){
+        return cy.get('[aria-label="'+userName+'"]')
+    }
+
+    getRole(role){
+        return cy.get('[aria-label="'+role+'"]')
+    }
+
+    getLoginFormForUser(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getLoginFormForRole(){
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getCalendar(){
```

Comment:
Is this still loginPage?

Reply:
Good point! Will move it to a different object. Thanks.